webextensions/_locales/fr/messages.json

```diff
@@ -133,250 +133,254 @@
+  "context_closeDescendants_label": { "message": "Fermer les descendants" },
+  "context_closeOthers_label": { "message": "Fermer les autres onglets sauf cet arbre" },
+  "context_collapseTree_label": { "message": "Replier cet arbre" },
+  "context_collapseAll_label": { "message": "Replier tout" },
+  "context_expandTree_label": { "message": "Déplier cet arbre" },
+  "context_expandAll_label": { "message": "Déplier tout" },
+  "context_bookmarkTree_label": { "message": "Ajouter cet arbre aux marque-pages…" },
+  "context_groupTabs_label":    { "message": "Créer un nouveau groupe depuis les onglets" },
+  "context_topLevel_prefix": { "message": "Élément au plus haut niveau : " },
 
-  "context_collapsed_label": { "message": "Plié (for testing of a checkbox type menu)" },
-  "context_pinnedTab_label": { "message": "Épinglé (for testing of a radio type menu)" },
-  "context_unpinnedTab_label": { "message": "Désépinglé (for testing of a radio type menu)" },
+  "context_collapsed_label": { "message": "Replié (pour tester un menu type case à cocher)" },
+  "context_pinnedTab_label": { "message": "Épinglé (pour tester un menu type bouton radio)" },
+  "context_unpinnedTab_label": { "message": "Désépinglé (pour tester un menu type bouton radio)" },
 
 
-  "config_title": { "message": "Tree Style Tab Options" },
+  "config_title": { "message": "Options de Tree Style Tab" },
 
   "config_appearance_caption": { "message": "Apparence" },
 
-  "config_sidebarPosition_caption": { "message": "Position du panneau des onglets:" },
+  "config_sidebarPosition_caption": { "message": "Style du contenu pour la position du panneau :" },
   "config_sidebarPosition_left": { "message": "À gauche" },
   "config_sidebarPosition_right": { "message": "À droite" },
-  "config_sidebarPosition_description": { "message": "*This configuration won't change the position of the sidebar itself. You need to click the switcher in the sidebar header and click \"Move Sidebar to Right\" manually." },
+  "config_sidebarPosition_description": { "message": "*Cette configuration ne changera pas la position du panneau latéral lui-même. Vous devez sur le bouton dans l'entête du panneau latéral et cliquer sur « déplacer le panneau latéral à droite » manuellement." },
 
-  "config_sidebarDirection_caption": { "message": "Alignement du titre des onglets:" },
-  "config_sidebarDirection_ltr": { "message": "À gauche" },
-  "config_sidebarDirection_rtl": { "message": "À droite" },
+  "config_sidebarDirection_caption": { "message": "Style pour la direction du texte :" },
+  "config_sidebarDirection_ltr": { "message": "De gauche à droite" },
+  "config_sidebarDirection_rtl": { "message": "De droite à gauche" },
 
   "config_style_caption": { "message": "Thème" },
-  "config_style_plain": { "message": "Plain" },
-  "config_style_plain_dark": { "message": "Plain Dark" },
+  "config_style_plain": { "message": "Simple" },
+  "config_style_plain_dark": { "message": "Simple sombre" },
```

Comment:
`Sombre simple`

Reply:
It’s plain but dark, not the other way round so I think it makes sense to me to add the «modifier» afterwards.